LocalSettings.php

```diff
@@ -4430,6 +4440,22 @@
 				'editmyoptions' => true,
 			),
 		),
+		'wikicanadawiki' => array(
```

Comment:
Well you would set it false if it were in LocalWiki. As I'm not sure on this I'm leaving someone else to merge.